test/unit/has_slug_tests.rb

```diff
@@ -150,6 +150,23 @@ class InitTests < UnitTests
       assert_equal exp,             subject.slug_db_column_value
     end
 
+    should "slug its source even if the source is already a valid slug" do
```

Comment:
@jcredding I'd probably add ``using `ardb_has_slug_generate_slug` `` to this test desc like the other tests.  Or remove that from all the tests - I'm fine either way.